### PR TITLE
Release note

### PR DESCRIPTION
**Pull request by VijayanB**
Add 1.1.0 release note

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release-notes/opendistro-odfe-cli-release-notes-1.1.0.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+* Allow non-secured cluster to be added to profile ([#41](https://github.com/opendistro-for-elasticsearch/odfe-cli/pull/41))
+
+### Infrastructure
+
+* Add release draft workflow ([#35](https://github.com/opendistro-for-elasticsearch/odfe-cli/pull/35))
+
+### Documentation
+
+* Add compatibility matrix section to ReadMe ([#31](https://github.com/opendistro-for-elasticsearch/odfe-cli/pull/31))
```
Comment (jmazanec15) [inline release-notes/opendistro-odfe-cli-release-notes-1.1.0.md:20]: nit: README

File: release-notes/opendistro-odfe-cli-release-notes-1.1.0.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* Add RFC for Rest API as commands. ([#42](https://github.com/opendistro-for-elasticsearch/odfe-cli/pull/42))
+* Add release guidelines ([#38](https://github.com/opendistro-for-elasticsearch/odfe-cli/pull/38))
+* Add doc reference as badge ([#46](https://github.com/opendistro-for-elasticsearch/odfe-cli/pull/46))
+* Update compatibility matrix ([#58](https://github.com/opendistro-for-elasticsearch/odfe-cli/pull/58))
+* Update ReadMe & Version ([#59](https://github.com/opendistro-for-elasticsearch/odfe-cli/pull/59))
```
Comment (jmazanec15) [inline release-notes/opendistro-odfe-cli-release-notes-1.1.0.md:25]: nit: README

Comment (VijayanB) [reply release-notes/opendistro-odfe-cli-release-notes-1.1.0.md]: Ack

File: release-notes/opendistro-odfe-cli-release-notes-1.1.0.md (hunk 1 above)
Comment (VijayanB) [reply release-notes/opendistro-odfe-cli-release-notes-1.1.0.md]: Ack

**Review by jmazanec15 — APPROVED** [review]

**Review by vamshin — COMMENTED** [review]
LGTM!

**Pull request merged**